### DISCUSSION TITLE
Wrong Buy Execution price

### DISCUSSION THREAD

**Discussion by Harsh-Maheshwari** [kernc/backtesting.py#416] [Q&A]
![Capture](https://user-images.githubusercontent.com/38434626/126315301-e4f6f177-375f-41b4-9dfa-03dba8e87726.PNG)
Somehow the buy price is outside the complete price range!
I am using self.buy() with all default values

**Comment by kernc** [comment]
The order is executed at _adjusted price_, which accounts for your set 3% commission.
